### PR TITLE
Added TTL

### PR DESCRIPTION
**Pull request by hanasu**

### CODE REVIEW

File: decode/lldp/ttl.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function ttl() {
+	this.ttl = undefined;
```
Comment (king-jam) [inline decode/lldp/ttl.js:2]: can remove this

File: decode/lldp/ttl.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function ttl() {
+	this.ttl = undefined;	
+}
+
+chassisId.prototype.decode = function(tlv, raw_packet, offset) {
+	this.ttl = raw_packet.readUInt16BE(offset+1, true);
```
Comment (king-jam) [inline decode/lldp/ttl.js:6]: change to tlv.ttlSeconds and there is no subType offset so just offset and not "offset+1"

**Comment by hanasu** [comment]
So like this instead?

function ttl() {
}

chassisId.prototype.decode = function(tlv, raw_packet, offset) {
    tlv.ttlSeconds = raw_packet.readUInt16BE(offset, true);
    offset++;
}

module.exports = ttl;

**Comment by king-jam** [comment]
no need to increment the offset with offset++

lldp.js takes care of traversing TLV to TLV. lldp_tlv.js increments the passed offset by 2 bytes to account for the TLV type and length fields. The offset inside this code is at the 0 index of the TTL data. 

Also didn't even notice the first time, chassisId.prototype.decode -> ttl.prototype.decode

**Pull request closed without merging**